Cargo.toml

```diff
@@ -3095,6 +3095,11 @@ lto = "fat"
 panic = "abort"
 
 [package.metadata.docs.rs]
-rustdoc-args = ["-Zunstable-options", "--cfg", "docsrs"]
+rustdoc-args = [
+  "-Zunstable-options",
+  "--cfg",
+  "docsrs",
```

Comment:
note that `--cfg docsrs` has been [removed](https://github.com/bevyengine/bevy/commit/8f5345573c49322dfa672c3328fdca4bf1881488) across the project!

Reply:
I see there _a lot_ of conflicts due to this :)

Reply:
Fixed!